components/MediList/index.js

```diff
@@ -18,36 +18,55 @@ export default function MediList() {
     setDosage('');
   };
 
+  const handleSearch = () => {
+    const filteredMedications = dummyMedications.filter(
```

Comment:
You probably want to display your medikament(s) here instead of your dummyMedications. This way a newly added medikament will be also displayed.

Reply:
for now i just want to show dummies. in the next user stories the new medications should be displayed